upgrade/scripts/upgrade/prerequisites.sh

```diff
@@ -188,13 +188,13 @@ if [[ $state_recorded == "0" && $(hostname) == "ncn-m001" ]]; then
     record_state ${state_name} "$(hostname)"
```

Comment:
i'd `tee` these ones to the log file as well cause this is what tells you a stage has been done. if we only record the start, we would need to guess whether it has been done or now

Reply:
Good idea!

Reply:
Done. Let me know how it looks to you now.